go-controller/pkg/kubevirt/router.go

```diff
@@ -166,33 +179,73 @@ func EnsureLocalZonePodAddressesToNodeRoute(watchFactory *factory.WatchFactory,
 		}); err != nil {
 			return fmt.Errorf("failed adding static route: %v", err)
 		}
+
 	}
 	return nil
 }
 
-// EnsureRemoteZonePodAddressesToNodeRoute will add static routes when live
-// migrated pod belongs to remote zone to send traffic over transwitch switch
-// port of the node where the pod is running:
-//   - A dst-ip with live migrated pod ip as prefix and nexthop the pod's
-//     current node transit switch port.
-func EnsureRemoteZonePodAddressesToNodeRoute(watchFactory *factory.WatchFactory, nbClient libovsdbclient.Client, pod *corev1.Pod, nadName string) error {
+func reconcileClusterDefaultNetworkLocalZoneNeighbours(watchFactory *factory.WatchFactory, pod *corev1.Pod, podAnnotation *util.PodAnnotation, nadName string) error {
+	// TODO: Implement neighbours update for non IC
```

Comment:
Is this something you are planning to do? Is there any upstream issue tracking it that we can link?

Reply:
We are planning to not support this for non IC, I don't think there are any live migration for default pod network non IC users.